test/functional/tests/stress/test_stress_without_io.py

```diff
@@ -0,0 +1,331 @@
+            cache.stop()
+            caches_count = len(casadm_parser.get_caches())
+            if caches_count != 0:
+                TestRun.fail(f"Expected caches count: 0; Actual caches count: {caches_count}.")
+            cores_count = len(casadm_parser.get_cores(cache.cache_id))
+            if cores_count != 0:
+                TestRun.fail(f"Expected cores count: 0; Actual cores count: {cores_count}.")
+        with TestRun.LOGGER.step("Reload CAS modules."):
+            cas_module.reload_all_cas_modules()
+        with TestRun.LOGGER.step("Load cache."):
+            cache = casadm.load_cache(cache.cache_device)
+            caches_count = len(casadm_parser.get_caches())
+            if caches_count != 1:
+                TestRun.fail(f"Expected caches count: 1; Actual caches count: {caches_count}.")
+            cores_count = len(casadm_parser.get_cores(cache.cache_id))
+            if cores_count != 1:
+                TestRun.fail(f"Expected cores count: 1; Actual cores count: {cores_count}.")
+        with TestRun.LOGGER.step("Validate cache configuration."):
+            if cache.get_cache_config() != cache_config:
+                TestRun.fail("Cache configuration is different than before reloading modules.")
+        TestRun.LOGGER.end_iteration()
+    TestRun.LOGGER.end_group()
+
+    with TestRun.LOGGER.step("Check md5 of test file."):
+        check_files(core, size_before_load, permissions_before_load, md5_before_load)
+    casadm.stop_all_caches()
+
+
+def check_files(core, size_before, permissions_before, md5_before):
+    TestRun.LOGGER.info("Checking file md5.")
+    core.mount(mount_point)
+    file_after = fs_utils.parse_ls_output(fs_utils.ls(test_file_path))[0]
+    md5_after = file_after.md5sum()
+    if md5_before != md5_after:
+        TestRun.LOGGER.error(f"Md5 before ({md5_before}) and after ({md5_after}) are different.")
+
+    if permissions_before.user != file_after.permissions.user:
+        TestRun.LOGGER.error(f"User permissions before ({permissions_before.user}) "
+                             f"and after ({file_after.permissions.user}) are different.")
+    if permissions_before.group != file_after.permissions.group:
+        TestRun.LOGGER.error(f"Group permissions before ({permissions_before.group}) "
+                             f"and after ({file_after.permissions.group}) are different.")
+    if permissions_before.other != file_after.permissions.other:
+        TestRun.LOGGER.error(f"Other permissions before ({permissions_before.other}) "
+                             f"and after ({file_after.permissions.other}) are different.")
+    if size_before != file_after.size:
+        TestRun.LOGGER.error(f"Size before ({size_before}) and after ({file_after.size}) "
+                             f"are different.")
+    core.unmount()
+
+
+def prepare_with_file_creation(config):
+    cache_dev, core_dev = prepare(config)
+    TestRun.LOGGER.info(f"Tested configuration: cache mode={config}")
```

Comment:
This is redundant to what pytest does for parametrization. If we want additional logging per iteration we can do this in conftest.py.